packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -442,6 +442,11 @@ export interface PluginModel {
         version: string;
     };
     entryPoint: PluginEntryPoint;
+    packageUri: string;
+/**
```

Comment:
please apply our formatting :) `Format Document` action in Theia or VS Code

Reply:
Oops. You're right. Done